### PR TITLE
Info texts

### PR DESCRIPTION
**Pull request by PatricKangasmaki**
Added missing tooltips. The ones in the settings didn't have specified texts yet so I left them to TBD for now.

### CODE REVIEW

File: frontend/src/pages/TimeEstimationPage.tsx [hunk 1]
```diff
@@ -110,12 +112,43 @@ export const TimeEstimationPage = () => {
...
+              },
+              {
+                header: t('Custom'),
+                text: t(
+                  'estimate with a custom value for work amount in working days.',
+                ),
+              },
+            ],
+          },
+        },
```
Comment (PatricKangasmaki) [inline frontend/src/pages/TimeEstimationPage.tsx:142]: I couldn't figure out how to pass react components as params to the modal so I had to do this little workaround where I pass the data as an object with the desired fields and handle it in the modal. This approach should work well with the other infomodals that follow the same design.

**Review by IidaKainu — CHANGES_REQUESTED** [review]
> This approach should work well with the other infomodals that follow the same design.

Since info modals so far follow the same design I agree

Few suggestions
+ I think TimeEstimationPage.module.scss .infoIcon should be changed to correspond with other planner info icons:
```
.infoIcon {
  margin-left: 12px;
}
```

File: frontend/src/pages/TimeEstimationPage.tsx (hunk 1 above)
Comment (juhoeerola) [reply frontend/src/pages/TimeEstimationPage.tsx:142]: Yeah, the modal payload must be serialisable to json, so components are not possible as params.

The solution here seems good 👍

**Review by juhoeerola — CHANGES_REQUESTED** [review]

**Comment by PatricKangasmaki** [comment]
Really great points, added them all 👍

**Review by juhoeerola — APPROVED** [review]

**Review by IidaKainu — APPROVED** [review]
👍 

> I think TimeEstimationPage.module.scss .infoIcon should be changed to correspond with other planner info icons

I also added a suggestion for this (not sure why it bothers me so much 😄). Feel free to discard the commit if not needed

**Comment by PatricKangasmaki** [comment]
> I also added a suggestion for this (not sure why it bothers me so much 😄). Feel free to discard the commit if not needed
Ah I forgot to add that part completely! Didn't skip it on purpose, I just went through each comment and added all the suggestions but missed this one while implementing them 😄

**Pull request merged**